### PR TITLE
Support for IBDesignable and Swift

### PR DESCRIPTION
**Pull request by matt-oakes**
Hello,

Thanks for the great little library, it's been invaluable with some recent projects.

I've made two small changes to the library with the intention of supporting the new features in Xcode 6 - namely Swift and IBDesignable.
- Added support for the `IBDesignable` annotation by specifying which bundle the NIB should be loaded from. It's a good assumption that the NIBs are located in the same bundle as the class itself.
- Also added support for Swift by making sure the module name is not considered when creating the default NIB name.

Thanks again for the library!

Matt

### CODE REVIEW

File: UIView+NibLoading.m [hunk 1]
```diff
@@ -18,7 +18,7 @@ + (UINib*) _nibLoadingAssociatedNibWithName:(NSString*)nibName
     UINib * nib = associatedNibs[nibName];
     if(nil==nib)
     {
-        nib = [UINib nibWithNibName:nibName bundle:nil];
+        nib = [UINib nibWithNibName:nibName bundle:[NSBundle bundleForClass:[NibLoadedView class]]];
```
Comment (n-b) [inline UIView+NibLoading.m]: Good point, but the `NibLoadedView` class may be in a different bundle than the current view class being loaded. (The subclass of `NibLoadedView`. We’d rather use `[NSBundle bundleForClass:self]`.

File: UIView+NibLoading.m [hunk 2]
```diff
@@ -87,7 +87,13 @@ - (void) loadContentsFromNibNamed:(NSString*)nibName
 
 - (void) loadContentsFromNib
 {
-    [self loadContentsFromNibNamed:NSStringFromClass([self class])];
+    NSString *classString = NSStringFromClass([self class]);
```
Comment (n-b) [inline UIView+NibLoading.m]: "className", please.

File: UIView+NibLoading.m [hunk 3]
```diff
@@ -87,7 +87,13 @@ - (void) loadContentsFromNibNamed:(NSString*)nibName
...
+    NSRange range = [classString rangeOfString:@"."];
+    if (range.location != NSNotFound)
+    {
+        classString = [classString substringFromIndex:range.location + range.length];
```
Comment (n-b) [inline UIView+NibLoading.m]: Very good point. I think this deserves a comment in the code (as well as the header, actually).

**Comment by n-b** [comment]
Hi Matt, thanks a lot for the PR and sorry for the delay. Both changes are very welcome, if you can make the small amends I mentioned in the comments.

(Also: these two changes are unrelated, each one would deserve its own PR. But that’s just me worrying about pointless procedure details.)

(And thanks again!)

**Comment by matt-oakes** [comment]
I've made the changes suggested. I've added a comment to the implementation as to why it's needing to search for a `.` to remove the module name. I left it out of the header as it feels more like an implementation detail, rather than something a user of the class would need to know about.

**Comment by n-b** [comment]
👍

**Comment by matt-oakes** [comment]
Thanks! Is it possible to do another release so this can be picked up through Cocoapods?

**Comment by n-b** [comment]
@matto1990 Done. (I think. I’m not a cocoapod user myself. Let me know if you see something fishy.)

**Comment by matt-oakes** [comment]
That seems to have done the trick! Thanks :dancer:

**Pull request merged**